docs/index.md

```diff
@@ -1,9 +1,46 @@
 
 # Welcome to WheelNext
 
-## Introduction
-
-Content to be added here.
+## What is WheelNext and Why should you participate !?
+
+WheelNext is an open-source initiative (<https://github.com/wheelnext> & <https://wheelnext.dev/>) aiming to improve the
+user-experience in the Python packaging ecosystem, specifically around the scientific computing and machine/deep learning space.
```

Comment:
```suggestion
user experience in the Python packaging ecosystem, specifically around the scientific computing and machine/deep learning space.
```

Reply:
I'd add something like "We also anticipate benefits in other domains that heavily rely on performance of compiled Python extension modules - the benefit of utilizing one's hardware more optimally is not exclusive to any single domain."

Reply:
Suggestion added and merged